build-libs/redirects.js

```diff
@@ -203,6 +203,12 @@ async function buildDevPortalRedirects() {
 			destination: '/waypoint',
 			permanent: true,
 		},
+		{
+			source: '/waypoint/docs/:slug',
+			destination:
+				'https://github.com/hashicorp/waypoint/tree/main/website/content/docs',
```

Comment:
Comments on [this ticket](https://app.asana.com/0/0/1206697393359298/f) has more info

Reply:
@heatlikeheatwave Could you provide some context on why this redirect was removed?

Reply:
Yes, definitely. Initially, I was going to fold in the redirect for `/waypoint/docs/<path>`, but the redirect needed more finessing to work properly, and I didn't want it to block merging this PR. It's also another a ticket in the project.

You can see the other PR [here](https://github.com/hashicorp/dev-portal/pull/2375) for that change.

Reply:
Ah, that makes sense! Thanks for the context 👍